builtin/providers/aws/resource_aws_db_subnet_group.go

```diff
@@ -82,12 +83,17 @@ func resourceAwsDbSubnetGroupRead(d *schema.ResourceData, meta interface{}) erro
 	}
 
 	if len(describeResp.DBSubnetGroups) != 1 ||
```

Comment:
I'm reading it as "if there is 0 or more than 1 :: OR :: the first one is not `d.Id()`". Seem right?

> Also, for readability assigning the 0-indexed group to a var might be nice after this check.

Legit